### PR TITLE
Updated mentions search logic

### LINKED ISSUE #102: Update mentions search logic to show all results
- [x] Update the logic of mentions to show all the results. Add a scrollbar to limit the overflow.
- [x] Sort all results in alphabetic order.
- [x] Update filter query to check substring instead of prefix.
- [x] Show "no results" if no results are found.

@AbhayVAshokan _a

### PR DESCRIPTION
**Pull request by AbhayVAshokan**
Fixes #102
- Moved the position of the color picker in the fixed menu.
- Alphabetically sorted all mentions.
- Updated filter logic to use "substring" instead of "startsWith".
- Added the message "no results" if the mentions query does not match anyone.
- Removed limit 5 from mentions logic, added a fixed width and height to mentions popup with overflow scroll.
- Added scroll handler for mentions popup (ref #82). Extracted the logic to a separate file.

Demo: https://vimeo.com/659636934/76ba15c94b

@amaldinesh7 _a please review.
cc: @labeebklatif

### CODE REVIEW

**Comment by amaldinesh7** [comment]
@AbhayVAshokan _a Please check this video https://vimeo.com/659865181/8e90ce7daa

**Review by amaldinesh7 — CHANGES_REQUESTED** [review]
@AbhayVAshokan I've added one comment. Please take a look.

File: lib/components/Editor/CustomExtensions/useCustomExtensions.js [hunk 1]
```diff
@@ -74,9 +74,16 @@ export default function useCustomExtensions({
       customExtensions.push(
         Mention.configure({
           suggestion: {
-            items: createMentionSuggestions(mentions, {
-              showImage: showImageInMention,
-            }),
+            items: createMentionSuggestions(
+              mentions.sort((mention1, mention2) => {
+                const name1 = mention1.label || mention1;
```
[the remark this answers is not in the document]
Comment (AbhayVAshokan) [reply lib/components/Editor/CustomExtensions/useCustomExtensions.js:79]: @amaldinesh7 `mention1` is either an `object` or a `string`.
```js
[
  {
    label: "Oliver Smith",
    key: "oliver-smith",
    imageUrl: "https://i.pravatar.cc/300",
  },
  "Jaden Smith",
]
```

If the above approach is confusing, I can change it to `if (typeof mention1 === "string") { ...`.

[the remark this answers is not in the document]
Comment (amaldinesh7) [reply lib/components/Editor/CustomExtensions/useCustomExtensions.js:79]: Yes. Please do explicitly specify either of the condition.

**Comment by AbhayVAshokan** [comment]
@amaldinesh7 _a please look at the new changes.

**Comment by amaldinesh7** [comment]
@AbhayVAshokan _a The issue mentioned in the video still persists. Did you look into that issue?

**Comment by AbhayVAshokan** [comment]
Oops. Let me take a look.

**Comment by AbhayVAshokan** [comment]
@amaldinesh7 _a please have a look at the changes. It was an issue specific to Chrome.
Mentions separator new line

**Comment by amaldinesh7** [comment]
LGTM!

**Pull request merged**